### DISCUSSION TITLE
Error Touchlink scan

### DISCUSSION THREAD

**Discussion by MirkoGr87** [Koenkk/zigbee2mqtt#22621] [Devices]
Buona Sera 
Sono Nuovo e ho riscontato un problema di questo tipo.
ogni volta faccio una scanzione per accoppiare mi da questi errori qualcuno mi può dare una risposta?

Info 2024-05-13 22:12:24Start Touchlink scan
Info 2024-05-13 22:12:24Set InterPAN channel to '11'
Warning 2024-05-13 22:12:25Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:25Set InterPAN channel to '15'
Warning 2024-05-13 22:12:25Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:25Set InterPAN channel to '20'
Warning 2024-05-13 22:12:26Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:26Set InterPAN channel to '25'
Warning 2024-05-13 22:12:27Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:27Set InterPAN channel to '12'
Warning 2024-05-13 22:12:27Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:27Set InterPAN channel to '13'
Warning 2024-05-13 22:12:28Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:28Set InterPAN channel to '14'
Warning 2024-05-13 22:12:28Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:28Set InterPAN channel to '16'
Warning 2024-05-13 22:12:29Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:29Set InterPAN channel to '17'
Warning 2024-05-13 22:12:29Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:29Set InterPAN channel to '18'
Warning 2024-05-13 22:12:30Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:30Set InterPAN channel to '19'
Warning 2024-05-13 22:12:30Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:30Set InterPAN channel to '21'
Warning 2024-05-13 22:12:31Scan request failed or was not answered: 'Error: Timeout - null - 254 - null - 4096 - 1 after 500ms'
Info 2024-05-13 22:12:31Set InterPAN channel to '22'
Info 2024-05-13 22:12:38Restore InterPAN channel
Error 2024-05-13 22:12:44Request 'zigbee2mqtt/bridge/request/touchlink/scan' failed with error: 'SRSP - AF - interPanCtl after 6000ms'

**Comment by averter** [comment]
Same issue here on version [1.38.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.38.0) commit: [f184730](https://github.com/Koenkk/zigbee2mqtt/commit/f184730)
